ideas/122-sob-increase_platform_usage.md

```diff
@@ -0,0 +1,60 @@
+## Preamble
+
+    Idea: 122-sob-increase_platform_usage
+    Title: Open Bounty - Increase Platform Usage
+    Status: Draft
+    Created: 2018-04-04
+    Related: 
+
+## Summary
+
+To identify the successful use of a platform, it's important and imperative to be able to measure its usage. As the Open Bounty platform continues to grow and evolve, its important to be able to track its growth, and make data-driven decisions.
+
+One of the key objectives of Open Bounty is to increase the usage of the platform. The purpose of this swarm is to build the tools and means necessary to capture these metrics and be able to report on them on a regular basis.
+
+## Swarm Participants
+
+- Lead Contributor\PM: @arash009 (Arash)
+- Contributor: @siphiuel (Vitaliy)
+- Testing and QA: 
+
+## Product Overview
+
+This swarm will create the mechanism to capture **three** specific metrics on the usage of Open Bounty (specific criteria for each metric yet to be defined):
+
+- **Value transacted** — This metric is to capture the **$USD equivalent** of total tokens that have been paid out in a given timeframe (daily, weekly, monthly, annual)
+- **Active contributors** — This metric is to capture the **Monthly Active Contributors** against each organization.
```

Comment:
I'd also like to see "Activation Metrics" being part of this. I.e. percentage of users who created/claimed a bounty within XX days after they signed up.

Reply:
**Active contributors** is the metric that measures the number of individuals who have claimed a bounty in past 30 days.

We don't have a metric that measures active bounty creators though, so that could be useful

Reply:
Right, so difference to activation metrics is the "within XX days after they signed up" bit.

Reply:
Maybe we can do that in later (semi-optional) iterations? Seems useful to keep scope tight. Don't need to decide this right now maybe?